rs/replicated_state/tests/replicated_state.rs

```diff
@@ -1009,18 +1009,6 @@ proptest! {
         }
 
         prop_assert_eq!(replicated_state.output_message_count(), 0);
-
-    }
-
-    #[test]
-    fn peek_next_loop_terminates(
```

Comment:
Is this dropped because some other test covers it?

Reply:
Yeah, there's a `peek_and_next_consistent()` test case above that does the same plus compares the two returned values for equality and checks the number of returned and remaining (0) messages.